charter.html

```diff
@@ -163,13 +163,16 @@ <h2>Scope</h2>
             The Working Group will specify new web platform features intended to
             be implemented in browsers or similar user agents. The purpose of
             these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+            privacy guarantees with a strong technical basis. To maintain privacy, new features will not enable
```

Comment:
Incorporating @npdoty's feedback, I would suggest we instead say:

The purpose of these features is to support web advertising in private ways. Here "private" refers to appropriate processing of personal information. Examples of ways in which new web platform features might enable inappropriate processing include (but are not limited to):
- enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-cross-site">cross-site recognition of users</a>
- enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-same-site">same-site recognition</a> of users across the clearing of state or data.

Reply:
+1 for explicitly marking these as examples. "appropriate processing of personal information" as a shorthand definition for private is necessarily still incomplete and imperfect, but certainly not the worst definition I've heard. :)

Reply:
@bslassey Do you object to making the above change?

Reply:
No objection, I think that's a great articulation of what we're trying to get across

Reply:
Changing `site` to `context` would make this acceptable under consensus.